spyder/config/manager.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Copyright © Spyder Project Contributors
+#
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+# ----------------------------------------------------------------------------
+
+""""""
+
+import os.path as osp
+
+from spyder.config.base import get_home_dir, SUBFOLDER
+from spyder.config.main import CONF_VERSION, DEFAULTS
+from spyder.config.user import UserConfig
+
+
+class ConfigurationManager:
+    """"""
+    _USER_CONFIG_LOCATION = None
+    _SYSTEM_CONFIG_LOCATION = None
+
+    def __init__(self):
+        """"""
+        self._parent = parent
+        self._site_config = None
+        self._project_config = None
+        self._global_config = None
+        self._project_configs = {}  # Cache project configurations
+
+        self.load_config()
+        self._remove_old_spyder_config_location()
+
+    def load_config(self):
+        # Main configuration instance
+        try:
+            self._global_config = UserConfig(
+                'spyder',
+                defaults=DEFAULTS,
+                load=True,
+                version=CONF_VERSION,
+                subfolder=SUBFOLDER,

```

Comment:
Perhaps we need to change `SUBFOLDER` in case there's an active project?

Reply:
Yes correct, I am just swapping the normal CONF for this manager and make sure all works as expected. Then I will add the correcto conf.

Just wanted to double check if this made sense